### ISSUE TITLE
Missing aria-label for inactive field button

### ISSUE THREAD

**Issue by BOHEUS** [twentyhq/twenty#8435]
**Scenario:**
1. Log in
2. Go to Settings > Data model > any object's details
3. Deactivate any field
4. Check in devtools rendered components

**Comment by harshrajeevsingh** [comment]
@Bonapara I already have a PR for this component. Can I add this to that?

**Comment by BOHEUS** [comment]
@harshrajeevsingh if you want to add component for this, it should be "Inactive {fieldName} button" or something along those lines, just make sure it's not generic like buttons in active fields

**Comment by harshrajeevsingh** [comment]
@BOHEUS No. There is no need to add component. What I meant was that I am already working on an issue where this aria label needs to be updated. Here it is: #8438

**Comment by BOHEUS** [comment]
You're right, I thought of aria-label as a component, not as an attribute
